spec/models/category_model_spec.rb

```diff
@@ -3,29 +3,18 @@
 require_relative '../spec_helper'
 
 RSpec.describe Category, type: :model do
-  DatabaseCleaner.strategy = :truncation
-
-  let(:user) { create(:user) }
-
-  after :each do
-    DatabaseCleaner.clean
-  end
-
   context 'with valid attributes provided' do
     it 'should create a new category with the logged in user' do
-      expect(user.categories.new(
-               category_name: 'Superman',
-               description: 'A world of heroes'
-             )).to be_valid
+      should validate_presence_of :category_name
```

Comment:
No need to change this one now... But in future, prefer `expect` rather than `should`. I think `should` has actually been deprecated. See [this on reasons why](https://github.com/rspec/rspec-expectations/blob/master/Should.md)